src/components/RangeSlider/components/DualThumb/DualThumb.scss

```diff
@@ -83,42 +92,56 @@ $range-thumb-shadow-focus: 0 0 0 rem(1px) color('indigo');
 .Thumbs {
   position: absolute;
   z-index: z-index('input', $stacking-order);
+  padding: 0;
   width: $range-thumb-size;
   height: $range-thumb-size;
   border-radius: 50%;
-  border: border-width() solid color('sky', 'lighter');
+  border: border-width() solid
+    var(--p-action-interactive, color('sky', 'lighter'));
   box-shadow: $range-thumb-shadow;
-  background: linear-gradient(color('sky', 'lighter'), color('sky', 'light'));
+  background: linear-gradient(
+    var(--p-action-interactive, color('sky', 'lighter')),
+    var(--p-action-interactive, color('sky', 'light'))
+  );
   -webkit-tap-highlight-color: transparent;
 
   // stylelint-disable-next-line value-no-vendor-prefix
   cursor: -webkit-grab;
+  transition-property: transform, margin;
```

Comment:
Is margin transitioning? If not you could just do something like this:
`transition: transform var(--p-duration-1-5-0, 0) var(--p-ease, linear)`

We currently don't have a 200ms duration, depending on how this looks maybe we can add it.